cranelift/filetests/src/zkasm_codegen.rs

```diff
@@ -63,6 +74,15 @@ pub fn generate_zkasm(wasm_module: &[u8]) -> String {
     program.join("\n")
 }
 
+fn handle_zkasm_settings(
```

Comment:
(just an option, no expectation to change anything)
I see the reasoning for having a dedicated function if we have more settings in the future, but at this stage, I would call this a premature optimization and follow YAGNI (You aren't gonna need it) [1], [2].

[1] https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it
[2] https://martinfowler.com/bliki/Yagni.html

Reply:
Adding a separate `handle_zkasm_settings` function also aims at encapsulating functionality and avoiding that `generate_zkasm(...)` grows too big. I think the separate function does not add any significant overhead here. No strong opinion on my side, merging now to wrap up the PR.